Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth
# Sorting Generic Artifact Files

#### Developer changelog:
- Changed a lot of icon names and icons location in order to not bloat artifact folder with a lot of generic icons and add new ones easier.


#### How to test:

- Check that I didn't broke artifact icons
- You can use `get_all_artifacts` in order to get all artifacts and check base artifact pictures
